dhall-kubernetes-generator/src/Dhall/Kubernetes/Convert.hs

```diff
@@ -226,6 +221,13 @@ toDefault definitions types modelName = go
       -- we get something unexpected
       _ -> error $ show modelName
 
+    -- | The imports that we get from the types are referring to the local folder,
```

Comment:
I'd prefer to keep definitions as local as possible, could we keep this definition in the `let` above instead of moving it here in the outer `where`?

Reply:
The function is called both here: https://github.com/dhall-lang/dhall-kubernetes/pull/69/files#diff-aa607019fd448aac4a0120a5458a96acR187 and here: https://github.com/dhall-lang/dhall-kubernetes/pull/69/files#diff-aa607019fd448aac4a0120a5458a96acR206 so i think it needs to be in the `where` if we want it to be visible to both

Reply:
Oh right sorry, I missed that 😄